### PR TITLE
fixed the CLI to exit with a non-zero status whenever deployment fails

### PR DESCRIPTION
**Pull request by tansengming**
Previously kumade would exit with 0 even with deployment failures

### CODE REVIEW

File: lib/kumade/cli.rb [hunk 1]
```diff
@@ -34,6 +34,7 @@ def self.swapping_stdout_for(io, print_output = false)
         rescue Kumade::DeploymentError
           io.rewind
           real_stdout.print(io.read)
+          exit 1
```
Comment (mike-burns) [inline lib/kumade/cli.rb:37]: Solid. I like this change.

File: lib/kumade/tasks/deploy.rake [hunk 2]
```diff
@@ -2,7 +2,7 @@ namespace :deploy do
   Kumade::Git.environments.each do |environment|
     desc "Deploy to #{environment} environment"
     task environment do
-      Kumade::CLI.run([environment])
+      Kumade::CLI.new([environment])
```
Comment (mike-burns) [inline lib/kumade/tasks/deploy.rake]: I don't understand this change.

Comment (tansengming) [inline lib/kumade/tasks/deploy.rake]: My bad, this should've been a separate pull request. Its to account for the fact that `cli.rb` no longer has `CLI.run`. I'm guessing all that code's been moved to the initializer instead.

Update: I've reverted changes to this file.

File: lib/kumade/cli.rb (hunk 1 above)
Comment (tansengming) [inline lib/kumade/cli.rb:37]: Thanks Mike. Scratched an itch that I had with Kumade.

**Pull request merged**